contrib/ansible-core/template.py

```diff
@@ -0,0 +1,39 @@
+pkgname = "ansible-core"
+pkgver = "2.17.3"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+]
+_runtime_deps = [
+    "python-cryptography",
+    "python-jinja2",
+    "python-packaging",
+    "python-resolvelib",
+    "python-pyyaml",
+]
+checkdepends = [
+    "git",
+    "openssh",
+    "python-pytest",
+    "python-pytest-mock",
+    "python-pytest-xdist",
+    *_runtime_deps,
+]
+depends = ["python", *_runtime_deps]
+pkgdesc = "Radically simple IT automation"
```

Comment:
this should be changed to concisely say what it does

and should be differentiated for core and the other package (i don't use ansible so i have no idea what the difference is, make it obvious)

Reply:
`Infrastructure as code software toolkit` + `subdesc = "core libraries"` :)

Reply:
I improvised something from Alpine's description that should be a bit better

Reply:
looks good :)